.env.development

```diff
@@ -0,0 +1,12 @@
+NEXT_PUBLIC_ENVIRONMENT = development
```

Comment:
Why does this file show up as new? Main already have these files and variables there. How do I see the diff with the latest main version?

Reply:
New main is not merged in this branch after the update of env pull request. I think that's why it is showing it as a new file.

Reply:
Can you merge new main to here so the reviewers can see the change?